Utils/logger_factory.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+
+class LoggerFactory:
+    @staticmethod
+    def create_logger(name, log_file_path = 'log.log', level = logging.INFO, log_to_file = True, log_to_console = True):
+        logger = logging.getLogger(name)
+        logger.setLevel(level)
+
+        # Formatter
```

Comment:
See what I wrote about the config_file_reader file regarding comments.

Here is literally a comment to a variable that has the same name. In code there is no need to separate sections of code by using comments most of the time.